analyses/SC_cd_2010/doc_SC_cd_2010.md

```diff
@@ -0,0 +1,23 @@
+# 2010 South Carolina Congressional Districts
+
+## Redistricting requirements
+In South Carolina, districts must:
```

Comment:
Can you add a link to any documents here? If it is old documents from links that are about to decay, you could potentially use waybackmachine